js/goto.js

```diff
@@ -9,6 +9,14 @@ function redirectUserToPage(page) {
 function redirectionHandler(data) {
   if (data.incompleteUserDetails) {
     redirectUserToPage(SIGNUP_URL);
+  } else if (
+    data.incompleteUserDetails === false &&
```

Comment:
you can do it like below 
!data.incompleteUserDetails && ! ...... likewise

Reply:
Can you create the separate variables for the roles like isDevelover, isMaven.... And then check?

Reply:
Done with changes. can you please check latest commit message